techmatters/flex-plugins
# CHI-1972 validate after case save

# Review after #1827 is merged

## Description

Clicking 'Add New Case' no longer checks if the form is valid before opening the new case.
Clicking 'Save and End' on the case now checks the contact form and bounces you back to the form with the usual error if it is not valid


### Checklist
- N/A Strings are localized

### Related Issues
CHI-1972

### Verification steps

1. Create an offline contact
2. Leave required form fields blank and click to 'Add New Case' - the new case should open for you to complete
3. Click 'Save and End' - you should see the invalid error and be put back to the contact form
